lib/puppet/provider/rabbitmq_cli.rb

```diff
@@ -40,6 +40,8 @@ def self.rabbitmq_version
     output = rabbitmqctl('-q', 'status')
     version = output.match(%r{RabbitMQ.*?([\d\.]+)})
     @rabbitmq_version = version[1] if version
+    return version[1] if version
+    return '3.8.1'
```

Comment:
The module’s default behavior is to install 3.6.x, IIRC. I’m not sure if line 43 is necessary, but pretty sure this is the wrong behavior.

Do you get this issue with the most recent release, or with git master?